packages/widget/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@synapsecns/widget",
   "description": "Widget library for interacting with the Synapse Protocol",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
revert change here for same reason Chi mentioned above

Reply:
fixed